internal/services/compute/windows_virtual_machine_resource.go

```diff
@@ -481,6 +496,32 @@ func resourceWindowsVirtualMachineCreate(d *pluginsdk.ResourceData, meta interfa
 		}
 	}
 
+	// only set patch assessment mode if it is defined in the config
+	if patchAssessmentMode != "" {
```

Comment:
but isn't there an implicit default value in the api?

Reply:
In private preview, removed this code.